autorelease/kokoro.py

```diff
@@ -42,13 +43,18 @@ def _send_pubsub_message(
     return resp
 
 
-def _make_build_request(job_name: str, sha: str, env_vars: dict = None) -> str:
+def _make_build_request(
+    job_name: str, sha: str, env_vars: dict = None, multi_scm_name: Optional[str] = None
```

Comment:
Would would it mean if a caller passed an empty string for multi_scm_name.  Would it mean the same as `None`?

To reduce this ambiguity, the declaration could be changed to:
`, multi_scm_name: str = ""`

Reply:
Done